keras_nlp/models/bert/bert_preprocessor.py

```diff
@@ -162,118 +168,50 @@ def __init__(
             sequence_length=sequence_length,
         )
 
-    @property
-    def tokenizer(self):
-        """The `keras_nlp.models.BertTokenizer` used to tokenize strings."""
-        return self._tokenizer
+    def call(self, x, y=None, sample_weight=None):
```

Comment:
Why did this move? Let's not generate unnecessary code delta, esp since I don't see any change.

This comment applies to all the other models as well.